test/utils/rpc.ts

```diff
@@ -40,6 +40,9 @@ export class RPC {
             method: 'POST',
             uri: `http://${this.host}:${this.port}`,
             body: payload,
+            headers: {
```

Comment:
are there other instances in the library where we are forming raw http requests outside of web3?

Reply:
Not for now.
But there will be on the other branch. Thanks for pointing this one out, cause now I'll not forget to refactor this one.